.travis.yml

```diff
@@ -25,6 +25,10 @@ matrix:
       env: TOXENV=py37
       dist: xenial
       sudo: true
+    - python: 3.8-dev
+      env: TOXENV=py
```

Comment:
Should this `py38`?

Reply:
Yeah, I tried py38, but apparently tox doesn't yet support py38.  But the docs do say by using "py" it will inherit a tox env for whatever Python interpreter is used.

https://tox.readthedocs.io/en/latest/example/basic.html#a-simple-tox-ini-default-environments

Reply:
I also opened an issue with tox: https://github.com/tox-dev/tox/issues/912